### PR TITLE
Backports 0.19 pr3

### PR DESCRIPTION
**Pull request by Munkybooty**

### CODE REVIEW

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
Looks good overall but pls redo 15696 (the code in question https://github.com/dashpay/dash/pull/4524#discussion_r733054282 was merged via https://github.com/dashpay/dash/pull/4484/commits/fce74ffd0c539baf5e9660f31782aa3e70923fc5) and squash 15840 fixes

**Comment by UdjinM6** [comment]
should redo 15949 - it should be applied _after_ 15696

File: src/wallet/rpcdump.cpp [hunk 1]
```diff
@@ -445,6 +446,13 @@ UniValue importpubkey(const JSONRPCRequest& request)
         throw std::runtime_error(
             RPCHelpMan{"importpubkey",
                 "\nAdds a public key (in hex) that can be watched as if it were in your wallet but cannot be used to spend. Requires a new wallet backup.\n"
+                "Hint: use importmulti to import more than one public key.\n",
+                {
+                    {"pubkey", RPCArg::Type::STR, /* opt */ false, /* default_val */ "", "The hex-encoded public key"},
+                    {"label", RPCArg::Type::STR, /* opt */ true, /* default_val */ "\"\"", "An optional label"},
+                    {"rescan", RPCArg::Type::BOOL, /* opt */ true, /* default_val */ "true", "Rescan the wallet for transactions"},
+                }}
+                .ToString() +
```
Comment (UdjinM6) [inline src/wallet/rpcdump.cpp:455]: 16071: drop these lines

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**